app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt

```diff
@@ -70,6 +71,7 @@ class AddSearchEngineFragment : Fragment(), CompoundButton.OnCheckedChangeListen
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        (activity as HomeActivity).getSupportActionBarAndInflateIfNecessary()
```

Comment:
Why can't we put this on `supportActionBar.show()` like the others? Is it because it's called in `onResume` after which the view might already be laid out?